### PR TITLE
Elemiegre patch 1

### PR DESCRIPTION
**Pull request by elemiegre**

### CODE REVIEW

**Review by vq783 — COMMENTED** [review]
MD works better with .md extension

**Review by vq783 — APPROVED** [review]
Fine!

File: evelien-kortrijk.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Kortrijk,Belgium
+
+The city is on the river Leie, 42 km (26 mi) southwest of Ghent and 25 km (16 mi) northeast of Lille
+
+## Restaurants
+
+Plenty of places to eat
```
Comment (githubteacher) [inline evelien-kortrijk.md]: Could you give me some examples? I like anything, but I ❤️ 🍣.

**Review by githubteacher — CHANGES_REQUESTED** [review]

**Review by Thubert1971 — CHANGES_REQUESTED** [review]
please change "garden" by "Garden" on line 18

File: evelien-kortrijk.md (hunk 1 above)
Comment (elemiegre) [reply evelien-kortrijk.md]: Sakura Japanese restaurant

**Review by Thubert1971 — APPROVED** [review]

**Comment by elemiegre** [comment]
dsfsdqf

**Pull request closed without merging**